src/components/linkButton/linkButton.module.css

```diff
@@ -1,9 +1,11 @@
 .button {
-  width: -webkit-fill-available;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
Have you considered adding `border-box` as a global default?

```
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}
```
(https://css-tricks.com/box-sizing/#aa-universal-box-sizing-with-inheritance)

Reply:
Good idea! I'll give it a test run to make sure it doesn't break any other components.

Reply:
I'm adding this as a separate task/PR.

Reply:
...because it's more a global fix and nor only related to hero